src/controllers/react.controller.js

```diff
@@ -18,10 +31,10 @@ const createReaction = catchAsync(async (req, res, next) => {
 });
 
 const getReaction = catchAsync(async (req, res, next) => {
-  const reactions = await React.find({ postId: req.params.postId });
-
+  const reactions = await React.find({ momentId: req.params.momentId }).populate('userId' , 'fullname avatar').sort({react : -1});
+  
```

Comment:
lấy momentId từ trên
```suggestion
  const { momentId } = req.params;
  const reactions = await React.find({ momentId }).populate('userId' , 'fullname avatar').sort({react : -1});
```
đoạn này check thêm đúng bài viết của mình mới cho xem còn không đúng => trả về fobidden

format lại code đi em